src/js/test/js.cpp

```diff
@@ -169,3 +173,264 @@ TEST_CASE("Check KV Map access")
     }
   }
 }
+
+bool str_contains(const std::string& s, std::string_view sv)
```

Comment:
Move to nonstd until https://cppreference.net/cpp/string/basic_string/contains.html?

Reply:
ok